src/System.Windows.Forms.Primitives/src/Interop/Shell32/Interop.IDragSourceHelper2.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Drawing;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.ComTypes;
+
+internal static partial class Interop
+{
+    internal static partial class Shell32
+    {
+        [ComImport]
+        [Guid("DE5BF786-477A-11D2-839D-00C04FD918D0")]
+        [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+        public interface IDragSourceHelper2
+        {
+            HRESULT InitializeFromBitmap(
+                SHDRAGIMAGE pshdi,
```

Comment:
```suggestion
                ref SHDRAGIMAGE pshdi,
```

https://docs.microsoft.com/en-us/windows/win32/api/shobjidl_core/nf-shobjidl_core-idragsourcehelper-initializefrombitmap

Reply:
Tests succeeded and you are awesome! Thank you!

Reply:
Accodring to the docs, the struct is passed by ref, i.e., `SHDRAGIMAGE* pshdi`